packages/react-components/src/utils/transformAlarmThresholdToSynchroChartsThreshold.ts

```diff
@@ -0,0 +1,62 @@
+import { YAnnotation } from '@iot-app-kit/core';
+import { AlarmData } from '../hooks/useAlarms';
+import { IoTEventsToSynchroChartsComparisonOperator } from '@iot-app-kit/source-iotsitewise';
+import { COMPARATOR_MAP, COMPARISON_OPERATOR } from '../common/constants';
+
+const createSynchroChartsThreshold = ({
+  thresholdValue,
+  severity,
+  labelText,
+  comparisonOperator,
+}: {
+  thresholdValue: number;
+  severity: number | undefined;
+  labelText: string;
+  comparisonOperator: COMPARISON_OPERATOR;
+}): YAnnotation => {
```

Comment:
Any reason we use `YAnnotation` here and not `Threshold` type? That's the type on `BarChartProps` and many other widgets.

Reply:
YAnnotation is the type from synchrocharts itself, if you look [here](https://github.com/awslabs/iot-app-kit/pull/3003/files/3d4be9a199498f310f1852fd331f88672aabbc92#diff-a635ab59949bd78e56473e361f379b8f7a3f9ec85d098ad7a75c2edb221a0eacR128) you can see we actually cast the barChart thresholds to this type